src/Apps/Auction/Routes/__tests__/ConfirmBid.test.tsx

```diff
@@ -107,13 +115,61 @@ describe("Routes/Register ", () => {
       }
     )
 
+    expect(mockPostEvent).toBeCalledWith({
+      action_type: Schema.ActionType.ConfirmBidSubmitted,
+      context_page: Schema.PageName.AuctionConfirmBidPage,
+      auction_slug: "saleslug",
+      artwork_slug: "artworkslug",
+      bidder_id: "bidderid",
+      bidder_position_id: "positionid",
+      sale_id: "saleid",
+      user_id: "my-user-id",
+    })
+    expect(mockPostEvent).toHaveBeenCalledTimes(1)
+
     expect(window.location.assign).toHaveBeenCalledWith(
       `https://example.com/auction/${
         ConfirmBidQueryResponseFixture.artwork.saleArtwork.sale.id
       }/artwork/${ConfirmBidQueryResponseFixture.artwork.id}`
     )
   })
 
+  it("send an error event to analytics if the mutation fails", async () => {
+    const env = setupTestEnv()
+    const page = await env.buildPage()
+
+    env.mutations.useResultsOnce(createBidderPositionFailed)
+
+    await page.agreeToTerms()
+    await page.submitForm()
+
+    expect(env.mutations.mockFetch).toHaveBeenCalledWith(
```

Comment:
Is this expectation needed? We stub [the mutation above](https://github.com/artsy/reaction/pull/2956/files#diff-97e6a9881aac57a032e2b138e21443fcR141), so while correct, it seems a bit extraneous to allow expect the mutation to be called, and increases the size of the test.

Reply:
We need it to confirm that we're correctly passing parameters to the GraphQL call.